BRANCHES.md

```diff
@@ -4,8 +4,9 @@ Each protocol version is specified in `Paper.tex` found in a branch of this repo
 
 | Branch            | Version                                                           | Applicable Block Numbers        |
 |-------------------|-------------------------------------------------------------------------------------------------------------------|-----------------------------------|
-| TBD               | [Gray Glacier](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/gray-glacier.md) <br> [Arrow Glacier](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/arrow-glacier.md) <br> [London](https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/london.md)        | Since 15,050,000 and onwards <br> Since 13,773,000 until 15,049,999 <br> Since 12,965,000 until 13,772,999 |
-| master            | [Berlin](https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/berlin.md)        | Since 12,244,000 until 12,964,999 |
+| TBD               | [Paris](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/gray-glacier.md)    | 15,537,394 until 17,034,869
+| london            | [Gray Glacier](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/gray-glacier.md) <br> [Arrow Glacier](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/arrow-glacier.md) <br> [London](https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/london.md)    |   Since 15,050,000 until 15,537,393 <br> Since 13,773,000 until 15,049,999 <br>Since 12,965,000 until 13,772,999 |
```

Comment:
I decided to point the `Arrow Glacier` and `Gray Glacier` to the same branch (london), as this was done for the `Muir Glacier` fork w.r.t `Istanbul`.

These could be separate PRs/branches, however.